utils/spacewalk-common-channels.ini

```diff
@@ -4203,16 +4203,16 @@ repo_url = https://repo.openeuler.org/openEuler-22.03-LTS/EPOL/main/%(arch)s
 archs	= x86_64, aarch64
 name	= Uyuni client tools for %(base_channel_name)s
 base_channels = openeuler2203-%(arch)s
-gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/EL8-Uyuni-Client-Tools/EL_8/repodata/repomd.xml.key
+gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/saltstack:/bundle:/next:/openEuler2203/openEuler_22.03/repodata/repomd.xml.key
```

Comment:
If we really need a separate bundle, then we'll need separate client tools at https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/openEuler2203-Uyuni-Client-Tools/EL_8 @raulillo82 (at the same for Master, of course).

Reply:
And @vzhestkov, I guess we'll need a fix at saltstack:bundle as well, not just next, so it can get linked?

Reply:
I think the last "EL-8" is extra, but yes, they don't exist yet. I wasn't very sure of what URL to write here, but creating this issue & PR would make things start :)

Reply:
`systemsmanagement:/Uyuni:/Stable:/EL8-Uyuni-Client-Tools/` is the project at OBS

`EL_8` is the repository inside that project. You can check the UI.

For openEuler, if a separate bundle is really needed, you will need to create the OBS projects for both Master and Stable, Marina can help with that.

And then here you will get something like 
https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/openEuler2203-Uyuni-Client-Tools/openEuler_22.03 for the URLs (or the equivalent for Master)

Reply:
@juliogonzalez given that `openEuler 22.03` was already  working using the EL9 client tools  repos, I don't think we need to create a  different set of  project, right? 
Or am I missing something? 
The unique doubt I have now that I'm  looking also at the prjconf is this:
https://build.opensuse.org/projects/systemsmanagement:saltstack:bundle:next:openEuler2203/prjconf
~~~
Prefer: open-sans-fonts

Macros:
%ext_man .gz
:Macros
~~~
that  I don't see in https://build.opensuse.org/projects/systemsmanagement:Uyuni:Master:EL8-Uyuni-Client-Tools/prjconf

Reply:
@raulillo82 should tell. From What I understood, EL8/9 bundles were not working and Ion created a separate bundle that would require a separate set of client tools.

Reply:
I believe that what we did was actually sent to the bundle in OBS, not that we had a special bundle. So the changes should already be there. This is the SR I sent: https://build.opensuse.org/request/show/1115041
But the best thing will be if @vzhestkov can confirm.

Reply:
Well, for now there is: https://build.opensuse.org/project/show/systemsmanagement:saltstack:bundle:next:openEuler2203, is this the bundle you used for testing?

If so, then https://build.opensuse.org/project/show/systemsmanagement:saltstack:bundle:openEuler2203 needs to be created, and we need a separate set of client tools.

Otherwise EL9 bundle can be used, and ttps://build.opensuse.org/project/show/systemsmanagement:saltstack:bundle:next:openEuler2203 should be removed.

Reply:
Yes, in the last tests (learning Tuesday, October), systemsmanagement:saltstack:bundle:next:openEuler2203 was used. However, yesterday with @deneb-alpha we were trying to see the differences and found nothing. If it is identical to the one for EL9, we wanted to save the hassle of maintaining yet another set of Uyuni tools. I can retest if the one from Uyuni:Stable:EL8-Uyuni-Client-ToolsEL_8 is still working.

Reply:
EL9 or EL8?

Reply:
I see differences in the prjconfs, but maybe they are not relevant
https://build.opensuse.org/projects/systemsmanagement:saltstack:bundle:next:openEuler2203/prjconf
https://build.opensuse.org/projects/systemsmanagement:saltstack:bundle:next:AlmaLinux8/prjconf

Still, I'd do a quick retest with EL8, if @vzhestkov confirms it makes sense using EL8. It could be the packages are building but there's son problem installing.

Reply:
But in any case, the URLs used right now, https://download.opensuse.org/repositories/systemsmanagement:/saltstack:/bundle:/next:/openEuler2203/openEuler_22.03/ are not right. We'll need either EL8 client tools, either new openEuler 22.03 client tools.

Reply:
Yes, in that case this PR would not be accepted and the code would be left as it is/was (so in the "files changed", left column would stay).

Reply:
Ok, but this PR can't be accepted as it's either, if EL8 bundle can't be used.

It's OK to have it as you have it now for testing.

So before merging, and if EL8 client tools can't be used, new client tools for openEuler will need to be created and used here.

Leaving this unapproved until this is clarified.

Reply:
Yes, this is exactly what I said: this PR will be discarded and things left as they are, with everything that was already merged in https://github.com/uyuni-project/uyuni/pull/6623